finetuner/labeler/__init__.py

```diff
@@ -27,25 +30,39 @@ def fit(
     :param train_data: Data on which to train the model
     :param clear_labels_on_start: If set True, will remove all labeled data.
     :param port_expose: The port to expose.
-    :param runtime_backend: The parallel backend of the runtime inside the Pea, either ``thread`` or ``process``.
+    :param runtime_backend: The parallel backend of the runtime inside the Pea, either
+        ``thread`` or ``process``.
     :param loss: Which loss to use in training. Supported
         losses are:
-        - ``CosineSiameseLoss`` for Siamese network with cosine distance
-        - ``EuclideanSiameseLoss`` for Siamese network with eculidean distance
-        - ``CosineTripletLoss`` for Triplet network with cosine distance
-        - ``EuclideanTripletLoss`` for Triplet network with eculidean distance
+        - ``SiameseLoss`` for Siamese network with cosine distance
+        - ``TripletLoss`` for Triplet network with cosine distance
+    :param preprocess_fn:
+    :param collate_fn:
     :param kwargs: Additional keyword arguments.
     """
     dam_path = tempfile.mkdtemp()
     stop_event = threading.Event()
 
+    # Remove all labels, as they are not needed
+    for doc in train_data:
+        try:
+            del doc.tags[__default_tag_key__]
+        except:
+            pass
```

Comment:
i would prefer a bit more defensive, not a fan of pass an exception sliently
```suggestion
    for doc in train_data:
        if doc.tags.get(__default_tag_key__):
            del doc.tags[__default_tag_key__]
```

Reply:
Check that your suggestions acutally work - the second one gives the error
```
  File "/home/tadej/miniconda3/envs/finetuner/lib/python3.9/site-packages/jina/types/mixin.py", line 97, in pop
    self._pb_body.ClearField(k)
ValueError: Protocol message has no "finetuner" field.
```